### PR TITLE
Document update

### PR DESCRIPTION
**Pull request by NiteshKoushik**
Change in key word "to" to "two"

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/elasticsearch.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ Name | Description
 ------------ | -------------
 Query | You can leave the search query blank or specify a lucene query
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/elasticsearch.md:190]:
```suggestion
Query | You can leave the search query blank or specify a lucene query.
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Nice catch! One more minor change, if you would. I spotted a missing period.

**Comment by NiteshKoushik** [comment]
Hello @oddlittlebird , Changes mad as per the suggestion.

**Review by oddlittlebird — APPROVED** [review]
Thanks for contributing to the docs!

**Pull request merged**